### DISCUSSION TITLE
What happened to the header and footer sections of the docs?

### DISCUSSION THREAD

**Discussion by Maestro132** [bootscore/bootscore#758] [Q&A] [answered]
I would always follow the documentation on Bootscore's website when starting a new site and now I need to create a favicon and site colours and this used to be described in the docs under header.php but that doesn't exist anymore. Why?

Anyways can someone please send me the instructions for creating a favicon and site colours. I remember uploading the logo to a website that generated all the required files.

**Comment by crftwrk** [comment]
Docs for header and footer have been moved to https://bootscore.me/documentation/templates/ for clarification because both files are just templates like all others. There you will find instructions for logo, favicons ands site color in the `header.php` list point.

**Reply by Maestro132** [reply]
Thanks for the info.

**Reply by Maestro132** [reply]
So I have just had the time to check and there are no instructions on generating the site colours and favicon. There used to be a link to a website that would generate the files and then we had to remove a line from one of the files and change the path or something like that. These instructions are missing unfortunately as shown in the screenshot.

![image](https://github.com/bootscore/bootscore/assets/98072860/1434715d-6f7d-4304-aff6-3e438a74f081)

**Reply by crftwrk** [reply, marked as answer]
Yes, Bootscore 6 arrived on Friday and this version has no favicons anymore because most users didn't setup this important part correctly. Since WordPress 6.5 you can choose favicon in Backend > Settings > General > Site Icon. So, icons are now part of WordPress and not part of the theme anymore. Docs have been changed to v6 with this release.

If you are on Bootscore v5, simply generate and replace icons using https://realfavicongenerator.net and change colors in `header.php`.
